x-pack/plugins/observability/public/utils/get_alert_evaluation_unit_type_by_rule_type_id.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export enum AlertEvaluationUnitType {
+  Duration,
+  Percent,
+  Number,
+}
+export const getAlertEvaluationUnitTypeByRuleTypeId = (
+  ruleTypeId: string
+): AlertEvaluationUnitType => {
+  switch (ruleTypeId) {
+    case 'apm.transaction_duration':
```

Comment:
Just thinking out loud:
Does it make sense that the Observability plugin knows about specific details of the APM rule types? I am wondering if this logic is something that should be provided by each app or if we should handle it like this in the Observability plugin. (Especially if we will have similar scenarios for each rule type)
@fkanout @simianhacker What do you think?

Reply:
@maryam-saeidi, it's a good question. The goal was to make the AlertSummary sharable, and then each App will import it and then use their logic/formatter.

However, when I started implementation, I realized that 
- APM (most likely other Apps) don't have formatters, AND they are importing and using Observability formatter. 
- Doing the formatting at the Rule level is not ideal, as we only have three types of values that need to be formatted across all rule types.

Reply:
Interesting. It makes sense that Observability shares formatter since the formatting logic can be re-used in different rule types. The concern here is to spread the logic related to rules in multiple plugins and make the Observability plugin unnecessarily aware of details of each rule type and as a result, in case of any change or addition, to always update this plugin as well. So basically, we are affecting the separation of concern here.